index.d.ts

```diff
@@ -1,32 +1,31 @@
 // TypeScript Version: 2.2
 import * as React from 'react';
 
-declare namespace DayPicker {
```

Comment:
I'm terribly sorry @adidahiya but I do not understand your recommendation on this point.

If I move `DayPicker` (and its DayPickerProps) to its own `DayPicker.d.ts`, I cannot import it in `index.d.ts`. It would have to be the other way around as `DayPickerProps` needs the definitions in index.d.ts.

Would it be possible to help me better understand the change that you would like me to do?

I did put back the `declare namespace DayPicker` with the `export = DayPicker;` at the end of the file. I tested it out in my sample Typescript app and it works. Coming from a C#/Java background, I would comment that it is a bit odd to see `DayPicker.DayPicker` but I'm relatively new to Javascript/Typescript so I'm open to leaving it back as it is.